migrate.go

```diff
@@ -151,7 +151,7 @@ func CollectMigrations(dirpath string, current, target int64) (Migrations, error
 	var migrations Migrations
 
 	// SQL migration files.
-	sqlMigrationFiles, err := filepath.Glob(dirpath + "/**.sql")
+	sqlMigrationFiles, err := filepath.Glob(dirpath + "/*.sql")
```

Comment:
> Just fix misleading code/comment. Not changed behaviour.

this looks like a change

Reply:
It seems that recursive pattern match has not been supported yet, and ** is equivalent to *.

FYI:
https://github.com/golang/go/issues/11862